averages.js

```diff
@@ -0,0 +1,18 @@
+class doesStats {
+  getTotal(nums) {
+    return nums.reduce(function(sum, val) {
+      return sum + val;
+    }, 0)
+    }
+  }
+
+  getMean(nums) {
+    return this.total(nums)/nums.length
```

Comment:
i don't believe `this.total` will be defined here since `this` refers to your constructor and `total` is not a function defined on `doesStats`